crates/sui-json-rpc/src/gateway_api.rs

```diff
@@ -141,6 +142,14 @@ impl RpcReadApiServer for GatewayReadApiImpl {
         Ok(self.client.get_object(object_id).await?)
     }
 
+    async fn get_past_object_maybe(
```

Comment:
if we move this method to FullNode api, then we don't have to implement this on the gateway. (that's the reason why we have ReadApi and FullNodeReadAPI, we will merge them when we can remove gateway)